src/scanning/scanning.lib.js

```diff
@@ -171,17 +171,20 @@ export const getThumbnailHeader = (type) => {
  * @param {string} instanceUrl
  * @param {Candidate} candidate
  * @param {string} type
- * @returns {string}
+ * @returns {string|null}
  */
 export function getThumbnailImageUrl(instanceUrl, candidate, type) {
   let thumbnail = candidate.thumbnails.find((t) => t.type === type);
   if (!thumbnail) {
-    throw new Error(`No thumbnail of type ${type} found`);
+    return null;
   }
   let res = thumbnail.public_url;
   if (type === "new" || type === "ref" || type === "sub") {
     res = instanceUrl + res;
   }
+  if (res.startsWith("http:")) {  // force https
```

Comment:
is that always true? Does that mean you can't test the mobile app with a local instance of skyportal (which usually doesn't have https)?